core/src/main/java/org/jdbi/v3/core/ConfigOnlyExtension.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core;
+
+import org.jdbi.v3.core.extension.ExtensionConfig;
+import org.jdbi.v3.core.extension.ExtensionFactory;
+
+class ConfigOnlyExtension<C extends ExtensionConfig<C>> implements ExtensionFactory<C>
```

Comment:
We could also have a config type for our reflective mappers, that would allow a global configuration of e.g. which `ColumnNameMappingStrategy` to use.